packages/ppx_factory/ppx_factory.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Nathan Rebours <nathan.p.rebours@gmail.com>"
+homepage: "https://github.com/cryptosense/ppx_factory"
+bug-reports: "https://github.com/cryptosense/ppx_factory/issues"
+license: "BSD-2"
+dev-repo: "git+https://github.com/cryptosense/ppx_factory.git"
+doc: "https://cryptosense.github.io/ppx_factory/doc"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+run-test: [
+  [ "dune" "runtest" "-p" name "-j" jobs ]
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```

Dune should not be tagged as build dei for forward compatibility reasons. Also from your dune-project file it looks like you need at least version 1.1